### DISCUSSION TITLE
Hazelcast supports memcache protocol

### DISCUSSION THREAD

**Discussion by sgandon** [grrolland/ngx-distributed-shm#16] [Q&A]
Hello dear owner,
I happened to find this page on hazelcasl
https://docs.hazelcast.com/imdg/latest/clients/memcache
which says it supports the memcache protocol since 3.12, I know that this project was created before Hazelcast 3.12 but I am wondering now if this is still relevant and if we could not use a plain Hazelcast server and the [memcache lua script](https://github.com/bungle/lua-resty-session/blob/master/lib/resty/session/storage/memcache.lua).
What do you think ?

**Comment by grrolland** [comment]
Hi,
The memcached protocol was supported when I began the project with 3.9.4 (see : https://docs.hazelcast.org/docs/3.9.4/manual/html-single/index.html#memcache-client) but don't work very well at this time, that's why I develop this projet, if fthe project is irrelevant today because Hazelcast directly do better, it's good ! 
Feel free to test on this new version. I think you'll have to adapt the lua script (this one should work for sessions : https://github.com/bungle/lua-resty-session/blob/master/lib/resty/session/storage/memcache.lua).

Kind Regard,
